exampleSite/content/docs/guide/configuration.md

```diff
@@ -137,3 +137,25 @@ copyright: "© 2023 YOUR TEXT HERE"
 ```
 
 For your reference, an example [`i18n/en.yaml`](https://github.com/imfing/hextra/blob/main/i18n/en.yaml) file can be found in the GitHub repository. Additionally, you could use Markdown format in the copyright text.
+
+## Add where appropriate:
+
+### Logo
+
+To modify the default logo, you'll need to place your logo .svg file under `/static/images` and name it as `logo.svg`.
+
+### Favicon
+
+To customize the favicon for your site, place your `favicon.ico` file (and favicon for other platform files) under `/static` and then add the following lines to your `hugo.yaml`:
```

Comment:
> To customize the favicon for your site, place your `favicon.ico` file (and favicon for other platform files) under `/static`

this is correct, and it's recommended to override all the files under: https://github.com/imfing/hextra/tree/main/static